File: 02775.cpp
#include <iostream>
using namespace std;

int main() {
	int T; cin >> T;

	for (int i = 0; i < T; i++) {
		int k, n; cin >> k >> n;

		int* room = new int[n + 1];
		for (int i = 0; i < n + 1; i++)
			room[i] = i;

		for (int i = 1; i <= k; i++)
			for (int j = 1; j <= n; j++)
				room[j] = room[j - 1] + room[j];

		cout << room[n] << endl;
		delete[] room;
	}
}

File: 02775.py
for i in range(int(input())):
    k = int(input())
    n = int(input())

    room = [i for i in range(n + 1)]
    for i in range(1, k + 1):
        for j in range(1, n + 1):
            room[j] = room[j - 1] + room[j]

    print(room[n])
